catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveCommandTest.java

```diff
@@ -91,6 +101,39 @@ protected SolrCacheMBean getCacheProxy() {
         verify(mbean, times(1)).removeById(idsArray);
     }
 
+    @Test
+    public void testOverMaxItemList() throws Exception {
+        ConsoleOutput consoleOutput = mock(ConsoleOutput.class);
+        int totalResultsToDelete = 100;
+        int resultsPerCFQuery = 22;
+        int numCFCalls = 5;
+        List<String> ids = new ArrayList<>();
+        List<Metacard> mcList = getMetacardList(totalResultsToDelete);
+        for(int i = 0; i < mcList.size(); i++) {
+            ids.add(mcList.get(i).getId());
+        }
+
+        RemoveCommand removeCommand = new RemoveCommand();
+
+        final CatalogFramework catalogFramework = mock(CatalogFramework.class);
+
+        QueryResponse queryResponse = mock(QueryResponse.class);
+        when(queryResponse.getResults()).thenReturn(getResultList(totalResultsToDelete));
+        when(catalogFramework.query(isA(QueryRequest.class))).thenReturn(queryResponse);
+
+        DeleteResponse deleteResponse = mock(DeleteResponse.class);
+        when(deleteResponse.getDeletedMetacards()).thenReturn(getMetacardList(resultsPerCFQuery));
+        when(catalogFramework.delete(isA(DeleteRequest.class))).thenReturn(deleteResponse);
+
+        removeCommand.catalogFramework = catalogFramework;
+        removeCommand.cqlFilter = "title like 'a*'";
```

Comment:
Consider using local variable to store this at the top of the method rather than passing in literal. Same with the boolean value below. May help with readability.